src/www/app/app.ts

```diff
@@ -589,14 +590,23 @@ export class App {
     }
   }
 
+  private isOutlineAccessKey(url: string): boolean {
+    if (!url) return false;
+
+    url = unwrapInvite(url);
+
+    return url.startsWith('ss://') || url.startsWith('ssconf://');
```

Comment:
Perhaps parse with the URL class and check the scheme instead, so it also validates that it's a URL.